reply-voting/config/routes.rb

```diff
@@ -0,0 +1,22 @@
+# @author: Aryan Ahuja
+# @created: 7/26/2021
+# 
+# @project: Reply Voting
+#
+# @description: defines routes, both POST and GET, and leads
+#				them to a specific action in ReplyVotingController
+
+Rails.application.routes.draw do
+	
+	#WARNING: EVERY vote_id passed in through the url will need to be UNIQUE
+
+	#this route, with /true, represents an upvote, adding a vote to the database
+	post '/reply/:replyid/:userid/:voteid/true' => 'replyvoting#addCount'
+
+	#this route, with /false, represents a downvote, removing a vote from the database
+	post '/reply/:relpyid/:userid/:voteid/false' => 'replyvoting#deleteCount'
```

Comment:
This should be a delete operation. 
`delete '/reply/:relpyid/:userid/:voteid/false' => 'replyvoting#deleteCount'`

Reply:
Either I'm overthinking this or I do not understand. If I do delete the route with `/false`, how would the front-end send a post request to remove a vote?

Reply:
the key word `delete` does not actually delete the route. This route will be triggered so when a user clicks the upvote or downvote for the second time. This will eliminate the need for passing in the boolean to determine if you are adding or deleting. This strategy would probably still work, so keep it for now. I will get back to you on this

Reply:
oh okay great! so I'm not making changes for now in this file